### DISCUSSION TITLE
Thoughts on making the 'async/await' blog a part of the official rust documentation.

### DISCUSSION THREAD

**Discussion by nihalpasham** [phil-opp/blog_os#1119] [General]
For folks who are new to rust and async/await, I think this [blog](https://os.phil-opp.com/async-await/) is perhaps the best introduction to the subject.  

I was wondering if there's been any interest in adding it to the official `async` rust-lang documentation?

**Comment by phil-opp** [comment]
Thanks a lot, great to hear that! I'm of course fine with contributing parts of that post to some official Rust docs if there is desire.

**Comment by nihalpasham** [comment]
I started a conversation [here](https://github.com/rust-lang/async-book/issues/186#issue-1530283800) on `async docs` repo.
